jest.config.js

```diff
@@ -12,9 +12,11 @@ module.exports = createConfig({
     '!packages/components/src/Accordion/useAccordionItem.ts',
     '!packages/components/src/Accordion/Accordion.tsx',
   ],
+  modulePathIgnorePatterns: ['<rootDir>/dist/'],
   moduleNameMapper: {
     '^@marigold/theme-(.+)$': '<rootDir>/themes/theme-$1/src',
     '^@marigold/(.+)-config$': '<rootDir>/config/$1',
     '^@marigold/(.+)$': '<rootDir>/packages/$1/src',
+    '^@marigold/(.*)$': '<rootDir>/dist/myscope/$1',
```

Comment:
Can we add a comment here why we do that, especially since "dist" is also on the ignore pattern 🫣

Reply:
But .... i still don't know why 😂🙈🙈🙈